src/main/kotlin/racing/Car.kt

```diff
@@ -0,0 +1,36 @@
+package racing
+
+import kotlin.random.Random
+
+class Car(val name: String) {
+
+    init {
+        require(name.length <= MAXIMUM_NAME_LENGTH) {
+            "자동차 이름은 ${MAXIMUM_NAME_LENGTH}자를 초과할 수 없다."
+        }
+    }
+
+    var progress = 0
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#class-layout

클래스 레이아웃에 따르면 프로퍼티가 더 위로 와야할 것 같아요!

Reply:
private set을 활용해서 외부에서 해당 프로퍼티를 수정하지 못하도록 막아보면 어떨까요?

```suggestion
    var progress = 0
        private set
```

Reply:
외부에서 접근 안되도록 하는 부분으로 수정하였습니다.